### PR TITLE
STY: unify imports in __init__.py

### PR DESCRIPTION
**Pull request by Amarnath1904**
improve code quality

### CODE REVIEW

**Review by eric-wieser — APPROVED** [review]
Seems harmless, although the PR title should explain what the change actually is

**Comment by Amarnath1904** [comment]
> Seems harmless, although the PR title should explain what the change actually is

hello, actually the  PR title explains what the change actually is I didn't make any big changes
all I did is a very small change is __inti_.py I think if you will see the code you will get it. :)

**Comment by Amarnath1904** [comment]
actually, this was my first PR so sorry if I don't make sense

File: numpy/__init__.py [hunk 1]
```diff
@@ -109,8 +109,7 @@
 import sys
 import warnings
 
-from ._globals import ModuleDeprecationWarning, VisibleDeprecationWarning
-from ._globals import _NoValue
+from ._globals import ModuleDeprecationWarning, VisibleDeprecationWarning , _NoValue
```
Comment (charris) [inline numpy/__init__.py:112]: Could you put this in a tuple? We try to keep lines < 80 characters. Something like
```
from ._globals import (
    ModuleDeprecationWarning, VisibleDeprecationWarning , _NoValue
)
```

Comment (eric-wieser) [reply numpy/__init__.py:112]: Of course, at this point it uses more lines than it did before...

Comment (rgommers) [reply numpy/__init__.py:112]: Still slightly better that way. Let's just get this in to not spend more time on style issues.

**Comment by rgommers** [comment]
Merged, thanks @Amarnath1904.

Please note that we prefer not to get pure style change PRs, because they're on average not worth the churn, review effort and CI time. If you want to try another PR, tackling a bug or documentation issue would be great.

**Pull request merged**